spec/models/user_spec.rb

```diff
@@ -12,22 +12,24 @@
   end
 
   it 'should connect to mailchimp if mailing list and confirmed' do
-    VCR.use_cassette('models/user_spec/mailing_list') do
-      # These tests are pretty meaningless with the VCR, but we also don't
-      # want to add fake sign-ups every time tests are run.
-      user = FactoryGirl.create(:confirmed_user)
-      user.mailing_list = true
-      user.save
-    end
+    stub_request(:post, "https://us8.api.mailchimp.com/2.0/lists/list")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.